problem_builder/public/js/mentoring_with_steps.js

```diff
@@ -217,11 +251,20 @@ function MentoringWithStepsBlock(runtime, element) {
         reviewButtonDOM.hide();
         tryAgainDOM.show();
 
+        /**
+         * We detach review step from DOM, this is required to handle HTML
+         * blocks that can be added to the Review step.
```

Comment:
@jbzdak "HTML blocks with embedded videos" perhaps? Also, it would be good to point out where the detaching happens - this comment had me confused for a second because L261 below doesn't do what it says.